playwright-test/multi-page/tests/forgot-password.spec.ts

```diff
@@ -0,0 +1,32 @@
+import type { ElementHandle } from '@playwright/test'
+import { test, expect } from './fixtures'
+
+test.describe('Forgot password page', () => {
```

Comment:
Should we added the `afterEach` to the tests too? I am just thinking once we add it to the docs and they come here to reference and wonder why we don't have the `afterEach` unless we communicate that it's we do it as part of `playwrightTest`

Reply:
No, [`@playwright/test` does it automagically](https://github.com/dequelabs/jazzband/blob/develop/integrations/javascript/src/playwrightTest.ts#L76-L78). It is not needed/included in docs.